src/loaders/lottie/tvgLottieProperty.h

```diff
@@ -178,6 +178,164 @@ static void _copy(PathSet* pathset, Array<PathCommand>& outCmds)
 }
 
 
+static bool _intersect(const Line& line1, const Line& line2, Point& intersection, bool& inside)
```

Comment:
`_modifier` seems inappropriate as a static internal function since it's accessed by another file, `LottieBuilder`. Introducing `LottieModifier.h/cpp` might be a better choice.